problems/5.loops.js

```diff
@@ -2,17 +2,29 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+function evenNumbers(num) {
+  let arr = [];
+  for(let i = 0; i <= num; i++) {
+    if(i % 2 === 0) {
```

Comment:
This is great. Another solution, since you're starting at 0 is to increment by 2 instead of one that way i is always even and you can skip the if statement.